pkg/platform/k8s/platform.go

```diff
@@ -19,98 +19,17 @@ import (
 	logrusr "github.com/bombsimon/logrusr/v4"
```

Comment:
Do we need this file?

Reply:
Currently yes because of `GetLabelsFromIP`.
This will be moved in a PR I'm just about to open.
So eventually yes - this file will be removed.